logos/logos-workernode/tests/test_vllm_process.py

```
@@ -251,6 +251,138 @@ def test_build_cmd_uses_default_chat_template_kwargs_flag(monkeypatch) -> None:
+@pytest.fixture()
+def chat_template_dir(monkeypatch, tmp_path: Path) -> Path:
+    """Relocate the persistent chat-template directory into a tmp_path."""
+    templates = tmp_path / "chat-templates"
+    templates.mkdir()
+    monkeypatch.setenv("LOGOS_CHAT_TEMPLATE_DIR", str(templates))
+    return templates
+
+
+def _handle_with_stub_binary(monkeypatch) -> VllmProcessHandle:
+    handle = VllmProcessHandle("lane-test", 19000, OllamaConfig())
+    monkeypatch.setattr(handle, "_resolve_vllm_binary", lambda _configured: "/tmp/vllm")
```

Ruff (0.16.1)

[error] Probable insecure usage of temporary file or directory: "/tmp/vllm"

(S108)